src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -3878,6 +3878,8 @@ def _get_kubelet_config(file_path):
         "containerLogMaxFiles", None)
     config_object.container_log_max_size_mb = kubelet_config.get(
         "containerLogMaxSizeMb", None)
+    config_object.pod_max_pids = kubelet_config.get(
```

Comment:
how do customer use the feature in cli?

Reply:
They would create a node config or kubelet config json file and apply it to the cluster as documented here: https://docs.microsoft.com/en-us/azure/aks/custom-node-configuration